adev/src/content/examples/aria/autocomplete/src/basic/app/app.component.ts

```diff
@@ -0,0 +1,111 @@
+import {Combobox, ComboboxInput, ComboboxPopupContainer} from '@angular/aria/combobox';
+import {Listbox, Option} from '@angular/aria/listbox';
+import {
+  afterRenderEffect,
+  ChangeDetectionStrategy,
+  Component,
+  computed,
+  ElementRef,
+  signal,
+  viewChild,
+} from '@angular/core';
+
+/**
+ * @title Autocomplete - Auto-Select Mode
+ */
+@Component({
+  selector: 'app-root',
+  templateUrl: 'app.component.html',
+  styleUrl: 'app.component.css',
+  imports: [Combobox, ComboboxInput, ComboboxPopupContainer, Listbox, Option],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class App {
+  popover = viewChild<ElementRef>('popover');
```

Comment:
```suggestion
  popover = viewChild<ElementRef<HTMLDivElement>>.required('popover');
```
Using `required ` will allow us to remove the type assertion